motion/ruby_motion_query/image.rb

```diff
@@ -17,6 +17,7 @@ class << self
 
       # @return [UIImage]
       def resource_for_device(file_base_name, opts = {})
+        file_base_name = file_base_name.gsub(/.png\z/, "")
```

Comment:
I wonder if we could just split on the extension.  So if you provided `file.jpg` then it would properly set `ext` to be `jpg` and also the same for PNG.

the only thing I can think of is someone naming images like "red.sky.green.grass.png" but is that really a valid concern?